File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/ciordems/d/EmsTransItemDO.java
package iih.ci.ord.ciordems.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 转科医嘱申请单 DTO数据 
 * 
 */
public class EmsTransItemDO extends OrCommonFieldsDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId_emsaptrans() {
		return ((String) getAttrVal("Id_emsaptrans"));
	}
	/**
	 * 主键
	 * @param Id_emsaptrans
	 */
	public void setId_emsaptrans(String Id_emsaptrans) {
		setAttrVal("Id_emsaptrans", Id_emsaptrans);
	}
	/**
	 * displaynam8
	 * @return String
	 */
	public String getId_or() {
		return ((String) getAttrVal("Id_or"));
	}
	/**
	 * displaynam8
	 * @param Id_or
	 */
	public void setId_or(String Id_or) {
		setAttrVal("Id_or", Id_or);
	}
	/**
	 * displaynam9
	 * @return String
	 */
	public String getId_orsrv() {
		return ((String) getAttrVal("Id_orsrv"));
	}
	/**
	 * displaynam9
	 * @param Id_orsrv
	 */
	public void setId_orsrv(String Id_orsrv) {
		setAttrVal("Id_orsrv", Id_orsrv);
	}
	/**
	 * 转科/跨科
	 * @return FBoolean
	 */
	public FBoolean getFg_tech_trans() {
		return ((FBoolean) getAttrVal("Fg_tech_trans"));
	}
	/**
	 * 转科/跨科
	 * @param Fg_tech_trans
	 */
	public void setFg_tech_trans(FBoolean Fg_tech_trans) {
		setAttrVal("Fg_tech_trans", Fg_tech_trans);
	}
	/**
	 * 目标科室id
	 * @return String
	 */
	public String getId_dep_to() {
		return ((String) getAttrVal("Id_dep_to"));
	}
	/**
	 * 目标科室id
	 * @param Id_dep_to
	 */
	public void setId_dep_to(String Id_dep_to) {
		setAttrVal("Id_dep_to", Id_dep_to);
	}
	/**
	 * 目标科室
	 * @return String
	 */
	public String getName_dep_to() {
		return ((String) getAttrVal("Name_dep_to"));
	}
	/**
	 * 目标科室
	 * @param Name_dep_to
	 */
	public void setName_dep_to(String Name_dep_to) {
		setAttrVal("Name_dep_to", Name_dep_to);
	}
	/**
	 * 目标病区
	 * @return String
	 */
	public String getName_dep_nur_to() {
		return ((String) getAttrVal("Name_dep_nur_to"));
	}
	/**
	 * 目标病区
	 * @param Name_dep_nur_to
	 */
	public void setName_dep_nur_to(String Name_dep_nur_to) {
		setAttrVal("Name_dep_nur_to", Name_dep_nur_to);
	}
	/**
	 * 目标病区id
	 * @return String
	 */
	public String getId_dep_nur_to() {
		return ((String) getAttrVal("Id_dep_nur_to"));
	}
	/**
	 * 目标病区id
	 * @param Id_dep_nur_to
	 */
	public void setId_dep_nur_to(String Id_dep_nur_to) {
		setAttrVal("Id_dep_nur_to", Id_dep_nur_to);
	}
	/**
	 * 转科原因
	 * @return String
	 */
	public String getDes_rea_canc() {
		return ((String) getAttrVal("Des_rea_canc"));
	}
	/**
	 * 转科原因
	 * @param Des_rea_canc
	 */
	public void setDes_rea_canc(String Des_rea_canc) {
		setAttrVal("Des_rea_canc", Des_rea_canc);
	}
	/**
	 * 原科室
	 * @return String
	 */
	public String getId_dep_from() {
		return ((String) getAttrVal("Id_dep_from"));
	}
	/**
	 * 原科室
	 * @param Id_dep_from
	 */
	public void setId_dep_from(String Id_dep_from) {
		setAttrVal("Id_dep_from", Id_dep_from);
	}
	/**
	 * 原病区
	 * @return String
	 */
	public String getId_dep_nur_from() {
		return ((String) getAttrVal("Id_dep_nur_from"));
	}
	/**
	 * 原病区
	 * @param Id_dep_nur_from
	 */
	public void setId_dep_nur_from(String Id_dep_nur_from) {
		setAttrVal("Id_dep_nur_from", Id_dep_nur_from);
	}
	/**
	 * 转科状态
	 * @return String
	 */
	public String getId_su_trans() {
		return ((String) getAttrVal("Id_su_trans"));
	}
	/**
	 * 转科状态
	 * @param Id_su_trans
	 */
	public void setId_su_trans(String Id_su_trans) {
		setAttrVal("Id_su_trans", Id_su_trans);
	}
	/**
	 * 转科状态编码
	 * @return String
	 */
	public String getSd_su_trans() {
		return ((String) getAttrVal("Sd_su_trans"));
	}
	/**
	 * 转科状态编码
	 * @param Sd_su_trans
	 */
	public void setSd_su_trans(String Sd_su_trans) {
		setAttrVal("Sd_su_trans", Sd_su_trans);
	}
	/**
	 * 申请时间
	 * @return FDateTime
	 */
	public FDateTime getDt_trans_apply() {
		return ((FDateTime) getAttrVal("Dt_trans_apply"));
	}
	/**
	 * 申请时间
	 * @param Dt_trans_apply
	 */
	public void setDt_trans_apply(FDateTime Dt_trans_apply) {
		setAttrVal("Dt_trans_apply", Dt_trans_apply);
	}
	/**
	 * 跨科生效时间
	 * @return FDateTime
	 */
	public FDateTime getDt_effe() {
		return ((FDateTime) getAttrVal("Dt_effe"));
	}
	/**
	 * 跨科生效时间
	 * @param Dt_trans_apply
	 */
	public void setDt_effe(FDateTime Dt_effe) {
		setAttrVal("Dt_effe", Dt_effe);
	}
	/**
	 * 跨科失效时间
	 * @return FDateTime
	 */
	public FDateTime getDt_end() {
		return ((FDateTime) getAttrVal("Dt_end"));
	}
	/**
	 * 跨科失效时间
	 * @param Dt_trans_apply
	 */
	public void setDt_end(FDateTime Dt_end) {
		setAttrVal("Dt_end", Dt_end);
	}
	/**
	 * 版本号
	 * @return FDateTime
	 */
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal("Sv"));
	}
	/**
	 * 版本号
	 * @param Sv
	 */
	public void setSv(FDateTime Sv) {
		setAttrVal("Sv", Sv);
	}
		/**
	 * 自费标识
	 * @return FBoolean
	 */
	public FBoolean getFg_selfpay() {
		return ((FBoolean) getAttrVal("Fg_selfpay"));
	}
	/**
	 * 自费标识
	 * @param Fg_selfpay
	 */
	public void setFg_selfpay(FBoolean Fg_selfpay) {
		setAttrVal("Fg_selfpay", Fg_selfpay);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/ciordems/d/EmsTransItemDO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;
using iih.ci.iih.ci.ord.ciordems.d;

namespace iih.ci.ord.ciordems.d
{
    /// <summary>
    /// EmsTransItemDO 的摘要说明。
    /// </summary>
    public class EmsTransItemDO : OrCommonFieldsDTO{

        public EmsTransItemDO() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id_emsaptrans {
            get { return getAttrVal<string>("Id_emsaptrans",null); }
            set { setAttrVal<string>("Id_emsaptrans", value); }
        }

        /// <summary>
        /// displaynam8
        /// </summary>
		public string Id_or {
            get { return getAttrVal<string>("Id_or",null); }
            set { setAttrVal<string>("Id_or", value); }
        }

        /// <summary>
        /// displaynam9
        /// </summary>
		public string Id_orsrv {
            get { return getAttrVal<string>("Id_orsrv",null); }
            set { setAttrVal<string>("Id_orsrv", value); }
        }

        /// <summary>
        /// 转科/跨科
        /// </summary>
		public bool? Fg_tech_trans {
            get { return getAttrVal<FBoolean>("Fg_tech_trans",null); }
            set { setAttrVal<FBoolean>("Fg_tech_trans", value); }
        }

        /// <summary>
        /// 目标科室id
        /// </summary>
		public string Id_dep_to {
            get { return getAttrVal<string>("Id_dep_to",null); }
            set { setAttrVal<string>("Id_dep_to", value); }
        }

        /// <summary>
        /// 目标科室
        /// </summary>
		public string Name_dep_to {
            get { return getAttrVal<string>("Name_dep_to",null); }
            set { setAttrVal<string>("Name_dep_to", value); }
        }

        /// <summary>
        /// 目标病区
        /// </summary>
		public string Name_dep_nur_to {
            get { return getAttrVal<string>("Name_dep_nur_to",null); }
            set { setAttrVal<string>("Name_dep_nur_to", value); }
        }

        /// <summary>
        /// 目标病区id
        /// </summary>
		public string Id_dep_nur_to {
            get { return getAttrVal<string>("Id_dep_nur_to",null); }
            set { setAttrVal<string>("Id_dep_nur_to", value); }
        }

        /// <summary>
        /// 转科原因
        /// </summary>
		public string Des_rea_canc {
            get { return getAttrVal<string>("Des_rea_canc",null); }
            set { setAttrVal<string>("Des_rea_canc", value); }
        }

        /// <summary>
        /// 原科室
        /// </summary>
		public string Id_dep_from {
            get { return getAttrVal<string>("Id_dep_from",null); }
            set { setAttrVal<string>("Id_dep_from", value); }
        }

        /// <summary>
        /// 原病区
        /// </summary>
		public string Id_dep_nur_from {
            get { return getAttrVal<string>("Id_dep_nur_from",null); }
            set { setAttrVal<string>("Id_dep_nur_from", value); }
        }

        /// <summary>
        /// 转科状态
        /// </summary>
		public string Id_su_trans {
            get { return getAttrVal<string>("Id_su_trans",null); }
            set { setAttrVal<string>("Id_su_trans", value); }
        }

        /// <summary>
        /// 转科状态编码
        /// </summary>
		public string Sd_su_trans {
            get { return getAttrVal<string>("Sd_su_trans",null); }
            set { setAttrVal<string>("Sd_su_trans", value); }
        }

        /// <summary>
        /// 申请时间
        /// </summary>
		public DateTime? Dt_trans_apply {
            get { return getAttrVal<FDateTime>("Dt_trans_apply",null); }
            set { setAttrVal<FDateTime>("Dt_trans_apply", value); }
        }
        /// <summary>
        /// 跨科生效时间
        /// </summary>
        public DateTime? Dt_effe
        {
            get { return getAttrVal<FDateTime>("Dt_effe", null); }
            set { setAttrVal<FDateTime>("Dt_effe", value); }
        }
        /// <summary>
        /// 跨科失效时间
        /// </summary>
        public DateTime? Dt_end
        {
            get { return getAttrVal<FDateTime>("Dt_end", null); }
            set { setAttrVal<FDateTime>("Dt_end", value); }
        }

        /// <summary>
        /// 版本号
        /// </summary>
		public DateTime? Sv {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }

        /// <summary>
        /// 服务内部编码
        /// </summary>
        public string Innercode_srvca
        {
            get { return getAttrVal<string>("Innercode_srvca", null); }
            set { setAttrVal<string>("Innercode_srvca", value); }
        }


        /// <summary>
        /// 费用同步标识
        /// </summary>
        public bool? Fg_syncfee
        {
            get { return getAttrVal<bool?>("Fg_syncfee", null); }
            set { setAttrVal<bool?>("Fg_syncfee", value); }
        }
        /// <summary>
        /// 自费标识
        /// </summary>
        public bool? Fg_selfpay
        {
            get { return getAttrVal<FBoolean>("Fg_selfpay", null); }
            set { setAttrVal<FBoolean>("Fg_selfpay", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[] { "Id_emsaptrans", "Id_or", "Id_orsrv", "Fg_tech_trans", "Id_dep_to", "Name_dep_to", "Name_dep_nur_to", "Id_dep_nur_to", "Des_rea_canc", "Id_dep_from", "Id_dep_nur_from", "Id_su_trans", "Sd_su_trans", "Dt_trans_apply", "Sv", "Fg_selfpay" };
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.ciordems.d.EmsTransItemDO";
        }
    }
}
